x/ibc/ante/ante.go

```diff
@@ -37,6 +37,9 @@ func (pvr ProofVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, sim
 			_, err = pvr.channelKeeper.AcknowledgePacket(ctx, msg.Packet, msg.Acknowledgement, msg.Proof, msg.ProofHeight)
 		case channel.MsgTimeout:
 			_, err = pvr.channelKeeper.TimeoutPacket(ctx, msg.Packet, msg.Proof, msg.ProofHeight, msg.NextSequenceRecv)
+		default:
```

Comment:
Interesting. Does this actually fix anything?

Reply:
no, but it prevents adding undesired events to non-IBC messages